src/ng1/ux-aspects-ng1.module.js

```diff
@@ -123,6 +123,7 @@ import './services/windowCommunicationService/windowCommunication.module.js';
 
 import './templates/angular-breadcrumb/angular-breadcrumb.html';
 import './templates/ui-bootstrap/pagination/pagination.html';
+import './directives/facets/facetOption/facetOptionDefault.html';
```

Comment:
I didn't want to create this file under /templates because it relates to the ng1 facets component, but I'm placing the import along with other templates because the import for the ng1 facets module doesn't include templates.